packages/eslint-plugin-office-addins/src/rules/no-context-sync-in-loop.ts

```diff
@@ -0,0 +1,30 @@
+import { TSESTree } from "@typescript-eslint/typescript-estree";
+
+export = {
+  name: "no-context-sync-in-loop",
+  meta: {
+    type: <"problem" | "suggestion" | "layout">"problem",
+    messages: {
+      loopedSync: "Calling context.sync() inside a loop can lead to bad performance",
+    },
+    docs: {
+      description: "Calling context.sync() inside of a loop dramatically increates the time the code runs the more iterations that are run",
```

Comment:
increates -> increases

Reply:
Thanks! 

. . . we really need a PM or documentation expert to chime in on the exact text.

Reply:
> Thanks!
> 
> . . . we really need a PM or documentation expert to chime in on the exact text.

Yeah, let's raise this concern on the daily, it would be great to have the documentation written the scearios